app/views/manuals/_manual_section.html.erb

```diff
@@ -1,6 +1,17 @@
-<div class='manual-body' id="content">
+<%
+  document_heading = []
+  document_heading << "#{presented_document.breadcrumb} - " if presented_document.breadcrumb.present?
+  document_heading << presented_document.title
+
+%><div class='manual-body' id="content">
   <article aria-labelledby="section-title">
-    <h1 id="section-title" class='section-title'><%= presented_document.title %></h1>
+    <%= render "govuk_publishing_components/components/heading", {
+      text: document_heading.join,
+      font_size: "m",
+      id: "section-title",
+      heading_level: 1,
+      margin_bottom: 4,
+    } %>
     <% if presented_document.summary %><p class='summary'><%= presented_document.summary %></p><% end %>
```

Comment:
🤔  Out of scope but noticed this appears as en empty tag

Reply:
Good catch - the `if` didn't account for `presented_document.summary` being an empty string, so adding `.present?` fixed this.